docs/sources/setup-grafana/configure-security/configure-authentication/azuread/index.md

```diff
@@ -169,6 +169,18 @@ GF_AUTH_AZUREAD_CLIENT_SECRET
 
 **Note:** Verify that the Grafana [root_url]({{< relref "../../../configure-grafana/#root-url" >}}) is set in your Azure Application Redirect URLs.
 
+### Access token expiration check and configure refresh token
+
+> Available in Grafana v9.3 and later versions.
+
+> **Note:** This feature is behind the `accessTokenExpirationCheck` feature toggle.
+
+Grafana verifies that the User's access token is not expired in case the User is logged in using an OAuth provider and it tries to obtain a new access token by using the provided refresh token (if any) if the access token is expired.
```

Comment:
nit (here and below): I'd break this sentence up into two parts to make it easier to read. Something like:
```suggestion
Grafana verifies that the User's access token is not expired in case the User is logged in using an OAuth provider. When the access token expires, Grafana tries to obtain a new access token by using the provided refresh token (if any).
```